overrides/resources/questbook/lang/en_us.lang

```diff
@@ -1,6 +1,6 @@
 # line 0
 nomifactory.quest.line.0.title=The Beginning
-nomifactory.quest.line.0.desc=GregTech Low Voltage Tier: start building a factory!%n%nUnlock access to §bDeep Mob Learning§r, early §bEnderIO§r tech, and §bApplied Energistics 2§r Digital Storage.%n%nBuild an §3Electric Blast Furnace§r for smelting advanced materials and alloys.
```

Comment:
This isn't the right file for QB edits, the new QB should go in `config/betterquesting/DefaultQuests.json`. (I need to clean this up)